frontend/app/modules/stock/stock-list-controller.js

```diff
@@ -0,0 +1,20 @@
+(function (angular) {
+  'use strict';
+
+  angular.module('stockModule').controller('StockListController', StockListController);
+
+  function StockListController(StockService) {
+    var vm = this;
+    StockService
+      .getAll()
+      .then(
+        function (response) {
+          vm.stockList = response.data._embedded.stocks;
```

Comment:
TODO: The controller should not know anything about a response or the format of the response, it should just get Stocks from the StockService.